funnel/templates/contacts.html.jinja2

```diff
@@ -72,7 +72,11 @@
                     {% for contact in daterow.contacts.all() %}
                       <div class="user">
                         <div class="user__box">
-                          <div class="user__box__gravatar user__box__gravatar--initials">{{ contact.participant.fullname|initials }}</div>
+                          {% if contact.participant.user and contact.participant.user.avatar %}
+                            <img class="user__box__gravatar" src="{{ contact.participant.user.avatar }}"/>
+                          {%- else %}
+                            <div class="user__box__gravatar user__box__gravatar--initials">{{ contact.participant.fullname|initials }}</div>
```

Comment:
Participant model should also get an avatar property that returns the user avatar if user is present. This if block can be removed then, replaced with `{{ user_avatar(participant) }}`